packages/ChangesReloaded-Core.package/CRBrowser.class/instance/createWindowSpecWith..st

```diff
@@ -5,4 +5,12 @@ createWindowSpecWith: aBuilder
 		model: self;
 		label: 'Changes Browser Reloaded';
 		extent: self extent;
+		children: (Set new
```

Comment:
What is the difference to using an `OrderedCollection` like our legacy guys did?

Reply:
It's now a `Set` and not an `OrderedCollection` :b  
Nahh, using an `OrderedCollection` might lead to the assumption that the order would matter - it does not, surprise, surprise! So we decided to use a `Set` to make this clear